py3status/modules/mysql_slave.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""
+Display MySQL/MariaDB slave state.
+
+This module displays the number of seconds the slave is behind the master.
+
+Configuration parameters:
+    cache_timeout: refresh cache_timeout for this module (default 5)
+    format_master: format used for master_servers
+                    (default '[\\?color=seconds {host} is master]')
+    format_slave: format used for slave servers
+                    (default '[\\?color=seconds {host} is {sec}s behind]')
```

Comment:
(QA) I like to avoid abbreviations in placeholders. `{sec} to {seconds}`. The new name would be consistent with the `seconds` threshold. (Q) Would it be useful to add new placeholders `{minutes}`, `{hours}` too with time conversion? 

You're missing `format`. Later, we can fix this by merging `format_master` and `format_slave` together.

Reply:
i don;t see the need for this, as normally a slave should not be behind the master ...
Maybe if someone else wants this we can add it later

Reply:
The comment was about the time conversio